apiserver/app/substitute.go

```diff
@@ -23,6 +23,21 @@ func (a *App) getUserSubstitutes(res http.ResponseWriter, req *http.Request) err
 	return nil
 }
 
+func (a *App) getUserOrganizations(res http.ResponseWriter, req *http.Request) error {
```

Comment:
I would rename it to `getUserOrganizationIds` we don't search for some organization metadata only for ID